Snakefile

```diff
@@ -1103,52 +1081,44 @@ rule prepare_sector_network:
         foresight=config["foresight"],
         water_costs=config["custom_data"]["water_costs"],
     input:
+        **branch(
+            sector_enable["land_transport"],
+            TRANSPORT,
+            {"dummy_transport": []},
+        ),
+        **branch(
+            sector_enable["heat"],
+            HEAT,
+            {"dummy_heat": []},
+        ),
```

Comment:
Just to be sure: are "dummy_transport" and "dummy_heat" entries really needed here? Can't be the third argument empty for `branch( )` function as it's the case for other it's calls?